packages/manager/modules/pci/src/projects/project/storages/databases/components/plans-list/plans-list.html

```diff
@@ -0,0 +1,83 @@
+<div class="row">
+    <oui-select-picker
+        class="d-inline-block col-md-6 col-xl-4 my-3"
+        data-ng-repeat="plan in $ctrl.plans track by plan.name"
+        data-disabled="$ctrl.currentPlan && $ctrl.plans.indexOf($ctrl.currentPlan) >= $index"
```

Comment:
Maybe should be in the controller ? 
Also isn't it risky to rely on index to check some capacities ?

Reply:
The code is now simpler and the comparison is more meaningful